### ISSUE TITLE
query: Primary key for elasticsearch

### ISSUE THREAD

**Issue by mithun12000** [yiisoft/yii2-elasticsearch#72]
Hey team,

I would like to know few thing..

1. why i need to call getPrimaryKey to get primary key everytime?
2. why i can't assign this primary key to mapped in Active Record so it will work fine with ```yii\grid\ActionColumn```?

I also have few observation.

1. In ```ActiveRecord```` search/find is calling ```mget``` method which will only set ```_id``` as primary key which is a ```Private member``` rather we can make it configuration so that we can assign this value to other value. 

method name ```populateRecord```
```
$pk = static::primaryKey()[0];//TODO should always set ID in case of fields are not returned
        if ($pk === '_id') {
            $record->_id = $row['_id'];
        }
```

2. If i also override ```static::primaryKey``` then ```mget``` will not be able to set primary key.

regards
Mithun Mandal

**Comment by beowulfenator** [comment]
I don't understand your question. Why is getting model's primary key using `getPrimaryKey` inconvenient? What is the problem with `ActionColumn`?

**Comment by beowulfenator** [comment]
Fixed by #110.
